text/0000-add-ember-page-title-addon.md

```diff
@@ -0,0 +1,47 @@
+- Start Date: 2020-06-24
+- Relevant Team(s): Ember CLI
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Tracking: (leave this empty)
+- Authors: Benjamin Jegard, Melanie Sumner, Ricardo Mendes
+
+# Add ember-page-title to the app blueprint
+
+## Summary
+
+Add [`ember-page-title`](https://github.com/adopted-ember-addons/ember-page-title) to the default blueprint for new Ember apps as a way to provide improved out-of-the-box (OOB) accessibility for Ember applications.
```

Comment:
does ember-page-title have a way to have a persistent leading dynamic segment? like if you want 
```
NullVoxPopuli | emberclear -- no unread
or
 
4 | NullVoxPopuli | emberclear

or

22 | emberclear // no active chat -- all unread

or

4 | NullVoxPopuli | 22 Unread total | emberclear
```

looking at the docs, it looks like, 

```
application route: 
```
{{page-title "emberclear"}}

{{#if (not (start-with this.router.currentRoute "chat"))}}
  {{page-title this.totalUnread " Unread total"
{{/if}}

```
chat/:id: 
```
{{page-title this.chattingWith.name}}

{{#if this.numUnread}}
  {{page-title this.numUnread }}
{{/if}}
```
?

Reply:
I'm not sure to totally understand the question. But I think the answer is yes 😅

Reply:
yeah, based on the docs, I think so, too :D